packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -352,30 +352,31 @@ const TileMoreOptions = React.forwardRef((props, forwardedRef) => {
   const [isMenuVisible, setIsMenuVisible] = React.useState(false);
 
   return (
-    <DropdownMenu
-      onShow={React.useCallback(() => setIsMenuVisible(true), [])}
-      onHide={React.useCallback(() => setIsMenuVisible(false), [])}
-      menuItems={(close) =>
-        children?.map((option: React.ReactElement) =>
-          React.cloneElement(option, {
-            onClick: (value: unknown) => {
-              close();
-              option.props.onClick?.(value);
-            },
-          }),
-        )
-      }
+    <Box
+      className={cx(
+        'iui-tile-more-options',
+        {
+          'iui-visible': isMenuVisible,
+        },
+        className,
+      )}
+      ref={forwardedRef}
+      {...rest}
     >
```

Comment:
Just wondering, why `<Box>` is now a parent of `<DropdownMenu>` instead of how it was earlier, a child?

Reply:
previously it was wrong.

dropdownmenu's children should almost always be a `<button>` instead of a plain `<div>`